src/core/shards/core.hpp

```diff
@@ -3379,6 +3379,7 @@ struct Once {
   SHTime current;
   SHTimeDiff next;
```

Comment:
Is this supposed to be an SHTimeDiff, next - current doesn't really make sense to me in that case.
next should also be just SHTime, no?

Reply:
std::chrono is a mess, everyone knows :) 
I just know it works lol
btw it's the same in "run" command mal side

Reply:
it's `diff` cos it's result of an operation. Also can't be SHTime in that case.

Reply:
Ah it's the same, `SHTimeDiff` is defined as `decltype(SHClock::now() - SHDuration(0.0))` which will be the same as `SHTime`